pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py

```diff
@@ -502,6 +502,19 @@ def houdini_parm_name(name):
             }
         ]
     },
+    {
+        'name': 'DisplayGamma',
+        'settings': [
+            {
+                'name': 'core:displayGamma',
+                'ui_name': 'Display Gamma',
+                'help': 'Display Gamma value',
```

Comment:
Maybe a bit more explanation, can we say something like:
"Gamma value, controls brightness of the display."